src/tools/crawling_tools.py

```
@@ -170,6 +173,128 @@ async def scrape(ctx: Context, url: str) -> str:
             )
             await ctx.report_progress(progress=80, total=100, message="Documents stored successfully")
             
+            # Store in Neo4j knowledge graph if enabled
+            neo4j_nodes_created = 0
+            kg_setting = os.getenv("USE_KNOWLEDGE_GRAPH", "false")
+            print(f"🔍 DEBUG: USE_KNOWLEDGE_GRAPH = '{kg_setting}'")
+            logger.info(f"🔍 USE_KNOWLEDGE_GRAPH = {kg_setting}")
```

Ruff (0.12.2)

Logging statement uses f-string

(G004)